src/components/BulkEnrollmentModal/index.jsx

```diff
@@ -71,8 +76,16 @@ class BulkEnrollmentModal extends React.Component {
         this.props.onSuccess();
       })
       .catch((error) => {
+        const { customAttributes } = error;
+        // fall back to the error.message instead of showing no information but prefer the
+        // cleaner info from customAttributes
+        const customErrMessage = truncateString(
+          customAttributes ? (customAttributes.httpErrorResponseData || error.message) : error.message,
+          240,
```

Comment:
What is this 240 doing?

Reply:
it’s the truncation limit, I’m still torn on truncating vs letting  it go as klong as it wants

Reply:
removed truncation per chat today in UX

Reply:
will leave function in there in case it's handy